corehq/apps/reports/dbaccessors.py

```diff
@@ -1,14 +1,14 @@
 import json
-import settings
-from corehq.apps.reports.models import HQExportSchema
+from django.conf import settings
 
 
-def get_exports(domain, include_docs=True, **kwargs):
+def _get_exports(domain, include_docs=True, **kwargs):
```

Comment:
I made this a private function, then the public functions can present a simpler set of parameters and they (or callers) can also handle wrapping.